migration/iD12/oracle/202404261129_IDEMPIERE-6129.sql

```diff
@@ -0,0 +1,4 @@
+-- IDEMPIERE-6129
+SELECT register_migration_script('202404261129_IDEMPIERE-6129.sql') FROM dual;
+
+ALTER TABLE t_selection_infowindow MODIFY value_string CLOB;
```

Comment:
oracle threw this error:
ORA-22858: invalid alteration of datatype

Maybe better first to try to increase it to a 4000 char?

Otherwise, better to conduct full tests in oracle because the change could affect the way how the String is saved and retrieved.